lib/hexpm_web/templates/blog/008-hex-v0.20-released.html.md

```diff
@@ -0,0 +1,48 @@
+## Hex v0.20 released
+
+<div class="subtitle">6 June, 2019 · by Eric Meadows-Jönsson</div>
+
+### Organization managed public packages
+
+A bit more than year ago we released private packages with organizations. With it you can add members to your organization and publish private packages only accessible by the members of the organization. The private packages are published to a separate repository (hosted by hex.pm) making sure they are separate from public packages.
+
+Now we are adding the ability to also manage public packages through your organizations. This means that an open source organization like [Phoenix Framework](/orgs/phoenixframework) or a company such as [Plataformatec](/orgs/plataformatec) can use Hex organizations to manage open source Hex packages just like private packages can be managed by organizations.
+
+When you first publish a new package you will be asked if you want to own it yourself or if it should be managed by an organization you are member of. An already published package can be transferred to an organization by running `mix hex.owner transfer PACKAGE ORGANIZATION`.
+
+Note that public packages managed by an organization does not change its namespace, all public packages still belong to the same global namespace, this feature is only changing the ownership of the package.
+
+### Diffing package versions
+
+With `mix hex.package diff` you can view the difference of two package versions from the command line. By calling `mix hex.package diff decimal 1.6.0..1.7.0` you will see a diff of the two versions. See the video below for examples on how you can combine it with `colordiff` and `less`.
```

Comment:
```suggestion
The new `mix hex.package diff` command shows the difference between two package versions directly in the command line. By calling `mix hex.package diff decimal 1.6.0..1.7.0` you will see a diff of the two versions. See the video below for examples on how you can combine it with `colordiff` and `less`.
```

Reply:
I don't know if "view ... from the command line" is correct so I rewrote it. Feel free to discard it if "view from" is actually ok!